dallinger/config.py

```diff
@@ -6,6 +6,7 @@
 import imp
 import logging
 import os
+from sets import Set
```

Comment:
Don't do the EMPTY_SET thing for the case where it sets self.synonyms in __init__, or you'll end up with all instances sharing the *same* set of synonyms.